### PR TITLE
made some changes from jquery to vanillajs

### PR DESCRIPTION
**Pull request by PruthviGBhat**

### CODE REVIEW

File: assets/script.js [hunk 1]
```diff
@@ -62,17 +63,20 @@ const getArchiveCards = i => {
...
-    $('body').removeClass('night')
+    $('body').classList.remove("night")
   }
-})
+});
 
 function demo() {
   setInterval(function() {
-    $('#toggle-box-checkbox').trigger('click')
+    var event = document.createEvent('HTMLEvents');
```
Comment (Syknapse) [inline assets/script.js:76]: on further examination I have just noticed that this piece of code is not doing anything in our project. Now that you are here you can have the pleasure of deleting the whole thing 😁  that's the full function called demo and it's body
```js
function demo() {
  setInterval(function() {
    $('#toggle-box-checkbox').trigger('click')
  }, 1000)
}
if (document.location.pathname.indexOf('fullcpgrid') > -1) {
  demo()
}
```

**Review by Syknapse — CHANGES_REQUESTED** [review]
This is a very nice contribution, helping us move away from jQuery. The changes requested are pretty simple, please implement them so we can merge your valuable contribution

**Comment by TenTraicion** [comment]
@PruthviGBhat, kindly make the requested changes so that we can merge your PR smoothly. for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

**Comment by Syknapse** [comment]
I'm reopening because these changes are useful for our code. If @PruthviGBhat does not want to make the improvements requested for us to merge, any of us maintainers can do it and merge it so we get the benefit of this PR

**Comment by TenTraicion** [comment]
Closing the PR as the updated PR is in work: #2077

**Pull request closed without merging**